docs/ares_parse_txt_reply.3

```diff
@@ -29,10 +29,8 @@ int ares_parse_txt_reply_ext(const unsigned char* \fIabuf\fP, int \fIalen\fP,
                              struct ares_txt_ext **\fItxt_out\fP);
 .fi
 .SH DESCRIPTION
-The
-.BR "ares_parse_txt_reply" " (" "ares_parse_txt_reply_ext" ")"
-function parses the response to a query of type TXT into a
-linked list (one element per sub-string) of
+The \fIares_parse_txt_reply(3)\fP function parses the response to a query of
+type TXT into a linked list (one element per sub-string) of
```

Comment:
looks like you removed reference to ares_parse_txt_reply_ext

Reply:
oops